src/osp/Active/SysNewton.cpp

```diff
@@ -311,20 +302,17 @@ void SysNewton::create_body(ActiveScene& rScene, ActiveEnt entity,
         // Get NewtonTreeCollision generated from elsewhere
         // such as SysPlanetA::debug_create_chunk_collider
 
-        if (entShape->m_collision)
+        if (entCollider->m_collision)
         {
             if (entBody.m_body)
             {
-                NewtonBodySetCollision(entBody.m_body, entShape->m_collision);
+                NewtonBodySetCollision(entBody.m_body, entCollider->m_collision);
             }
             else
             {
                 entBody.m_body = NewtonCreateDynamicBody(nwtWorld,
-                                                         entShape->m_collision,
-                                                         Matrix4().data());
+                    entCollider->m_collision, Matrix4().data());
```

Comment:
.data() from a default-constructed Matrix4 ? I feel like this warrents a comment elaborating.